wandb/sdk/internal/sender.py

```diff
@@ -295,6 +297,8 @@ def __init__(
 
         # job builder
         self._job_builder = JobBuilder(settings)
+        self._job_seq_id = None
+        self._job_version_alias = None
```

Comment:
why it's not part of the job_builder attributes?

Reply:
Good question, I will move them there